parlai/chat_service/services/terminal_chat/run.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Terminal Chat Runner.
+"""
+
+from parlai.core.params import ParlaiParser
+from parlai.chat_service.services.terminal_chat.terminal_manager import TerminalManager
+from parlai.chat_service.services.messenger import shared_utils as utils
+
+
+SERVICE_NAME = 'terminal'
+
+
+def setup_args():
+    """
+    Set up args.
+
+    :return: A parser that takes in command line arguments for chat services (debug, config-path, password), and a port.
+    """
+    parser = ParlaiParser(False, False)
+    parser.add_parlai_data_path()
+    parser.add_terminal_chat_args()
```

Comment:
again i feel better about not leaking these args into global parlai parser

Reply:
`chat_services/services/websocket/run.py` does a similar thing but with `parser.add_websocket_args()` so we just followed that. Should we just be using `add_chatservice_args` and getting the port number some other way?

Reply:
@stephenroller is saying you can add args locally here rather than define the function `add_terminal_chat_args` here, i suppose we could have done that as well with the websockets